tests/st/policy/test_profile.py

```diff
@@ -60,6 +60,15 @@ def setUp(self):
         # Get the original profiles:
         output = host1.calicoctl("get profile -o yaml")
         self.original_profiles = yaml.safe_load(output)['items']
+
+        # Filter out the default-allow profile
+        temp = []
+        for profile in self.original_profiles:
+            if profile['metadata']['name'] != "projectcalico-default-allow":
+                temp.append(profile)
+                    
+        self.original_profiles = temp   
```

Comment:
This bit of code saves copy of the profiles before tests executed. That original copy of the profiles is restored after a test. In this case, we just remove the default-allow profile from the original list.